tests/unit/components/form-test.js

```diff
@@ -56,6 +39,102 @@ describeComponent(
       ])
     })
 
+    let component, onChangeSpy, onValidationSpy, sandbox
+
+    beforeEach(function () {
+      sandbox = sinon.sandbox.create()
+      component = this.subject()
+      onChangeSpy = sandbox.spy()
+      onValidationSpy = sandbox.spy()
+
+      run(() => {
+        component.setProperties({
```

Comment:
Just wanted to mention that you can initialize property with `this.subject` as well.